climada/engine/supplychain.py

```diff
@@ -0,0 +1,316 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU Lesser General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Define the SupplyChain class.
+"""
+
+__all__ = ['SupplyChain']
+
+import logging
+import datetime as dt
+from pathlib import Path
+from tqdm import tqdm
+import numpy as np
+import pandas as pd
+from iso3166 import countries_by_numeric
+from climada.util import files_handler as u_fh
+from climada.engine import Impact
+from climada.util.constants import SYSTEM_DIR
+from climada.entity.exposures.base import Exposures
+
+LOGGER = logging.getLogger(__name__)
+WIOD_FILE_LINK = 'http://www.wiod.org/protected3/data16/wiot_ROW/'
```

Comment:
The url should go into the config file.

Reply:
done in https://github.com/CLIMADA-project/climada_python/pull/180/commits/ed468290bf322d343c230549529e51c09832e9d0